examples/config_file/endlessh-go.yml

```diff
@@ -0,0 +1,42 @@
+# Connection type. Possible values are tcp, tcp4, tcp6
+conn_type: "tcp"
```

Comment:
yml is fine.

I also want to hear opinion about `.env` style file, just like you are declaring environment variables.

```
ENABLE_PROMETHEUS=false
```

What do you think?